core/src/main/java/com/nuodb/migrator/cli/run/CliSchemaJob.java

```diff
@@ -116,43 +115,4 @@ protected ResourceSpec parseOutputGroup(OptionSet optionSet, Option option) {
         }
         return resource;
     }
-
-    @Override
-    protected Group createTargetGroup() {
```

Comment:
Is this test broken your change?

Reply:
@rshaull: I can't actually tell what this is testing, so I don't know if we are losing any test coverage by removing it. I'll defer to you.